mesh/tags/views.py

```diff
@@ -35,3 +35,35 @@ def user_tags(request):
         response_data.update({"status": "Success", "tags": user_tags})
 
         return JsonResponse(response_data, status=200)
+
+
+def post_tags(self, request):
+    if request.method == "POST":
+        data = request.POST
+        required_fields = ["tagName", "isDefault", "accountID"]  
+        if not all(field in data for field in required_fields):
+            response_data = {'status': "Error: Missing field"}
+            return JsonResponse(response_data, status=400)
```

Comment:
Have you tested your code with Postman? When I tried making a post request, I'm unable to get a response when there are no fields.

Reply:
I've tested the postman but I just created my own test cases. So I will get back to on this one